pyproject.toml

```
@@ -10,7 +10,7 @@ keywords = [
 license = "Apache 2.0"
 
 [tool.poetry.dependencies]
-python = "<3.11,>=3.7.1"
+python = "<3.12,>=3.7.1"
```

Thank you for putting in this PR. 

Since we're here could we just support up to Python 3.12 so 
```suggestion
python = "<3.13,>=3.7.1"
```